workspace/api/openfdd_bridge/zone_temp_analytics.py

```
@@ -299,6 +326,128 @@ def _recovery_rates(
+        sum(x["overnight_fan_on_minutes"] for x in weekday_nights) / len(weekday_nights)
+        if weekday_nights
+        else None
+    )
+    avg_weekend_night = (
+        sum(x["overnight_fan_on_minutes"] for x in weekend_nights) / len(weekend_nights)
+        if weekend_nights
+        else None
+    )
+    return {
+        "fan_column": fan_col,
+        "occupied_hours_local": f"weekdays {occ_start:02d}:00–{occ_end:02d}:00",
```

Ruff (0.15.15)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)